tests/worker.test.ts

```diff
@@ -7,48 +7,58 @@ import * as os from 'os'
 import { LogLevel } from '../src/types'
 
 jest.mock('../src/sql')
-jest.setTimeout(300000) // 300 sec timeout
+jest.setTimeout(600000) // 600 sec timeout
```

Comment:
Pretty long time, could we make this test take up less time somehow?

Reply:
The test actually takes closer to 2min than 10min. but when I did some heavy testing locally it went over the 5min mark. I incremented this just to not have to manually change it next time I run heavier tests. I think it's fine to keep as is. Anything longer than that has surely gotten stuck.